docs/tech-reference/distributor.rst

```diff
@@ -78,7 +78,9 @@ Supported keys
 ``repo-registry-id``
  The name that should be used for the repository when it is served by Crane. If specified
  it will be used for the ``repository`` field in the :ref:`redirect file <redirect_file>`.
- If a value is not specified, then repository id is used.
+ If a value is not specified, then repository id is used. Docker requires that this field
+ contains only integers, hyphens, and periods. Additionally a single slash can be used
```

Comment:
Good catch. I copied this from the help text and missed the first word. It actually has be lower case letters, docker doesn't accept upper case.